aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -289,6 +291,27 @@ Tensor _s_where(const Tensor& condition, const Tensor& self, const Tensor& other
   return ret;
 }
 
+void zero_numel_tensor_resize(Tensor& result, Tensor& result_indices, const Tensor& self, int64_t dim,
```

Comment:
I feel like a brief comment explaining what this function does and when it should be called would be helpful

Reply:
Done.